spec/spec_helper.rb

```diff
@@ -9,8 +9,5 @@
 
 require 'mocha/api'
 
-config = YAML.load_file(File.dirname(File.expand_path(__FILE__)) + "/test_connection.yml")
+ActiveRecord::Base.establish_connection({"adapter"=>"vertica5", "encoding"=>"utf8", "host"=>"localhost", "port"=>5433, "username"=>"dbadmin", "database"=>"docker"})
```

Comment:
This could definitely use a cleanup, but I just couldn't get circle to recognize a `test_connection.yml` file, not sure why

Reply:
I think it's just fine for now.